server/db/driver/pg/accounts.go

```diff
@@ -43,6 +45,34 @@ func (a *Archiver) Account(aid account.AccountID) (*account.Account, bool, bool)
 	return acct, isPaid, isOpen
 }
 
+// Accounts returns data for all accounts. Byte array fields in the database are
+// encoded as hex strings.
+func (a *Archiver) Accounts() ([]*db.Account, error) {
+	stmt := fmt.Sprintf(internal.SelectAllAccounts, a.tables.accounts)
+	rows, err := a.db.Query(stmt)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var accts []*db.Account
+	for rows.Next() {
+		a := new(db.Account)
+		id, pubkey, feeCoin := []byte{}, []byte{}, []byte{}
```

Comment:
These don't actually need to be allocated - `Scan` will do it again anyway - so, `var id, pubkey, feeCoin []byte` is good here.